l10n_it_fatturapa_out_rc/views/invoice_it_template.xml

```diff
@@ -17,12 +17,13 @@
                     t-set="rc_supplier"
                     t-value="invoices.mapped('rc_purchase_invoice_id.partner_id')"
                 />
-                        <t
-                    t-if="rc_supplier"
-                    t-set="partner_id"
-                    t-value="rc_supplier"
-                />
-                        <t t-if="rc_supplier" t-set="company_id" t-value="" />
+                        <t t-if="rc_supplier">
+                        <t t-set="partner_id" t-value="rc_supplier" />
+                        <t t-set="company_id" t-value="" />
+                    </t>
+                        <t t-if="not rc_supplier">
+                        <t t-set="partner_id" t-value="company_id.partner_id" />
+                    </t>
```

Comment:
Non ho problemi col diff attuale, ma quello minimo sarebbe:
```
diff --git a/l10n_it_fatturapa_out_rc/views/invoice_it_template.xml b/l10n_it_fatturapa_out_rc/views/invoice_it_template.xml
index 5b5046e8b..0a93e3297 100644
--- a/l10n_it_fatturapa_out_rc/views/invoice_it_template.xml
+++ b/l10n_it_fatturapa_out_rc/views/invoice_it_template.xml
@@ -13,6 +13,7 @@
                     <t
                 t-call="l10n_it_fatturapa_out.account_invoice_it_cedente_prestatore"
             >
+                        <t t-set="partner_id" t-value="company_id.partner_id" />
                         <t
                     t-set="rc_supplier"
                     t-value="invoices.mapped('rc_purchase_invoice_id.partner_id')"
```

Reply:
@TheMule71 grazie della review. Guarda se come l'ho modificato ti piace di più :wink:

Reply:
Per me va bene. Non c'è gran differenza tra avere il `t-if` in un tag separato o averlo all'interno dei due tag.  Diciamo che a parte il  `<t t-set="partner_id" t-value="company_id.partner_id" />` che mancava (il bug era lì, perché vai a rimpiazzare un template che contiene quel t-set con uno che non lo contiene), il resto del codice era comunque corretto, e la modifica è al limite solo estetica, e non significativa. E ci sono pro e contro in entrambi gli approcci.